Nighttime-Operations/Auxiliary-Telescope/AT-Pre/Prepare-for-On-Sky-Operations.rst

```diff
@@ -0,0 +1,112 @@
+
+- The :ref:`daytime checkout <AT-Daytime-Checkout-Daytime-Checkout>` has been executed successfully. 
+
+- The telescope has been venting (link to venting once merged) for a considerable amount of time. 
+  If venting wasn't possible, this procedure will take the system from the :ref:`calibration <statesLATISS-Combined-Calibrations-Procedure-LATISS-Combined-Calibrations-Generation-Procedure>` 
+  state or any other initial state to be prepared for on-sky observations. 
+
+.. _Open-for-On-Sky-Operations-Post-Condition:
+
+Post-Condition
+==============
+
+- Telescope is ready for on-sky operations. 
+  
+  Around 15 minutes before the end of the evening nautical twilight, proceed with the :ref:`beginning of the night procedure to center, align and focus the telescope. <AT-On-sky-WEP>`
+
+.. _Open-for-On-Sky-Operations-Procedure-Steps:
+
+Procedure Steps
+===============
+#. Review procedure :ref:`prerequisites <Open-for-On-Sky-Operations-Prerequisites>` and confirm they have been executed. 
+
+#. Announce in the #summit-announce and #summit-auxtel Slack channel that AuxTel is preparing to go on-sky. 
+
+#. Command the telescope and dome to prepare for on-sky observations. 
+
+   Load the ``auxtel/prepare_for/onsky.py`` script from the LOVE ``ATQueue`` panel, under ``AVAILABLE SCRIPTS`` by clicking on the blue icon. 
+
+   In the ``configuring script:onsky`` screen that pops up, leave the ``CONFIG`` box empty. 
+
+   .. figure:: ./_static/PrepareforOnSky_AuxTel.png
+     :name: prepareforonsky_AuxTel
+    
+     Screenshot of LOVE launching ``prepare_for/onsky.py`` script. 
+
+   The script includes the following steps:
+       - Slew telescope to park position. 
+       - If primary mirror cover is open, the script will now close it. 
+         This step is to protect the mirror and avoid particles and dust falling on it when the dome shutter opens.
+       - Move dome to oppose setting Sun to avoid direct sunlight inside the dome that might create thermal issues. 
+       - Open dome main shutter.
+       - Open primary mirror cover and vents. 
+       - Activate AOS open loop corrections.
+       - Enable dome following. 
+
+#. If the wind speed is below 8 m/s, manually open vent gate #3 using the remote controller and turn on the extraction fan power to 20 % from the dome pier.  
+   If wind speed is above or close to 8 m/s, keep vent gates and extraction fan off. 
+
+#. Visually confirm in LOVE displays:
+        - Telescope is in parking position; az = 90, el = 80 deg, rot = 0. 
+        - M1 Mirror cover is open. 
+        - ATAOS corrections are enabled. 
+        - Dome shutter is open and pointing towards the east; az = 90 deg. 
+     
+   
+   .. figure:: ./_static/PrepareforOnSky_AuxTel_LOVEdisplayConfirmation.png
+     :name: Confirmation of execution of ``prepare_for/onsky.py`` script LOVE 
+     
+     LOVE displaying AuxTel telescope and dome prepared for on-sky observations. 
```

Comment:
Yes, I was reluctant to include the names of the dashboards as some of these displays may change name and even arrangement.

Reply:
Can you link the dashboard instead of adding a name? So long as the name isn't baked into the link then that might help the issue of having the name get changed.